src/GroupRow.tsx

```diff
@@ -42,6 +42,7 @@ function GroupedRow<R, SR>({
   gridRowStart,
   groupBy,
   toggleGroup,
+  isRowSelectionDisabled,
   ...props
```

Comment:
Spreading props can be dangerous sometimes. Wonder if we should avoid it if possible